exercises/protein-translation/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: protein-translation
+version: 1.0.0.0
```

Comment:
given that https://github.com/exercism/problem-specifications/tree/master/exercises/protein-translation/canonical-data.json does not exist (that link will lead to a 404), the version number that is in accordance with the versioning policy (https://github.com/exercism/haskell/issues/522) will be 0.1.0.1 rather than 1.0.0.0